Neos.Neos/Classes/Fusion/ImageUriImplementation.php

```diff
@@ -129,6 +129,16 @@ public function getAsync()
         return $this->fusionValue('async');
     }
 
+    /**
+     * Async
+     *
+     * @return string|null
+     */
+    public function getFormat()
```

Comment:
Again, return type declaration, please.

Reply:
I did not add those because no other method on there used return types. Added them now

Reply:
Boyscouting, you know… Thanks!